src/controls/TouchControls.cpp

```diff
@@ -131,7 +131,8 @@ void TouchControls::renderPressedState(Renderer *renderer, UIAction action, bool
         .height = 6};
     renderer->fill_circle(x_circle, 15, 5, 0);
     renderer->flush_area(box);
-    renderPressedState(renderer, last_action, false);
+    // TODO - this causes a stack overflow when select is picked
```

Comment:
Good catch!
But I don’t get it, what a small partial update with do that just with this new save state addition?
Anyways that’s just a silly visual UX effect we could completely disable it.

Reply:
It's calling the same function that it's in - so it ends up calling itself again and again until there's no more stack space.